### PR TITLE
Integrated the jdbc smart driver with Sql* sample apps.

### PR DESCRIPTION
**Pull request by kneeraj**
Added a command line option for Sql* workloads called '--load_balance'.
By default it will be 'true'. But can be switched off by providing _--load_balance false_ in the command line arg
Modified the pom.xml to point to Yugabyte JDBC driver.
Ran the Sql payloads and seems to be working as expected.

### CODE REVIEW

File: src/main/java/com/yugabyte/sample/apps/AppBase.java [hunk 1]
```diff
@@ -179,6 +179,16 @@ protected Connection getPostgresConnection(String database, String username, Str
         ContactPoint contactPoint = getRandomContactPoint();
         Properties props = new Properties();
         props.setProperty("user", username);
+        // Check if the smart driver is there in the class path
+        boolean smartDriver = true;
+        try {
+          Class.forName("com.yugabyte.ysql.ClusterAwareLoadBalancer");
+        } catch (ClassNotFoundException cnfe) {
```
Comment (tedyu) [inline src/main/java/com/yugabyte/sample/apps/AppBase.java:186]: There are more than one exception which may be thrown from forName().

https://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#forName(java.lang.String)

Should the other exceptions be covered ?

File: src/main/java/com/yugabyte/sample/common/CmdLineOpts.java [hunk 2]
```diff
@@ -740,6 +753,8 @@ public static CmdLineOpts createFromArgs(String[] args) throws Exception {
     options.addOption("password", true,
         "The password to use when connecting to the database. " +
             "If this option is set, the --username option is required.");
+    options.addOption("load_balance", true,
+            "Set up YugabyteDB JDBC driver with in-built load balancing capability  ");
```
Comment (tedyu) [inline src/main/java/com/yugabyte/sample/common/CmdLineOpts.java:757]: nit: description should end with period.

File: src/main/java/com/yugabyte/sample/apps/AppBase.java (hunk 1 above)
Comment (kneeraj) [reply src/main/java/com/yugabyte/sample/apps/AppBase.java:186]: The other two exceptions are not necessary to be handled.
'LinkageError': As there is no dependency on any other class for ClusterAwareLoadBalancer to be initialised.
'ExceptionInInitializerError' - No static initialiser block in the class.

**Comment by tedyu** [comment]
LGTM

File: src/main/java/com/yugabyte/sample/apps/AppBase.java [hunk 3]
```diff
@@ -173,12 +173,29 @@ static boolean sleep(int durationMs) {
...
+      Class.forName("com.yugabyte.Driver");
+    } else {
+      Class.forName("org.postgresql.Driver");
+    }
     do {
       try {
         ContactPoint contactPoint = getRandomContactPoint();
         Properties props = new Properties();
         props.setProperty("user", username);
+        // Check if the smart driver is there in the class path
```
Comment (ashetkar) [inline src/main/java/com/yugabyte/sample/apps/AppBase.java:187]: nit: comment does not seem relevant to the code that follows.

File: src/main/java/com/yugabyte/sample/apps/SqlDataLoad.java [hunk 4]
```diff
@@ -312,6 +312,9 @@ private Key setupParams(PreparedStatement statement) throws Exception {
                 "--num_foreign_key_table_rows " + appConfig.numForeignKeyTableRows,
                 "--num_consecutive_rows_with_same_fk " + appConfig.numConsecutiveRowsWithSameFk,
                 "--batch_size " + appConfig.batchSize,
-                "--num_threads_write " + appConfig.numWriterThreads);
+                "--num_threads_write " + appConfig.numWriterThreads,
+                "--load_balance " + appConfig.loadBalance,
+                "--topology_keys " + appConfig.topologyKeys,
+                "--debug-driver " + appConfig.enableDriverDebug);
```
Comment (ashetkar) [inline src/main/java/com/yugabyte/sample/apps/SqlDataLoad.java:318]: Should it be `--debug_driver` instead of `--debug-driver`?
